docs/cloud/satellites/byoc/byoc.md

```diff
@@ -0,0 +1,88 @@
+# Bring Your Own Cloud
+
+"Bring Your Own Cloud" (BYOC) satellites **Experimental** are a hybrid between [self hosted](../self-hosted.md) and [earthly-managed](../../satellites.md) satellites. These satellites are managed by Earthly; just like our managed offering, but within your infrastructure. This blends the ease-of-use of managed satellites with the security benefits that self hosting can bring.
+
+BYOC satellites are [available with an Enterprise plan](https://earthly.dev/pricing).
+
+### Earthly BYOC Satellites vs Earthly Cloud Satellites
+
+| Similarities                                                                               | Differences                                                           |
+|--------------------------------------------------------------------------------------------|-----------------------------------------------------------------------|
+| ✅ The ability to use `earthly satellite` commands to provision and decommission satellites | ❌ It is up to you to get traffic to and from the satellite            |
+| ✅ Earthly-provided monitoring and reliability                                              | ❌ Earthly cannot access the machines for debugging or troubleshooting |
+| ✅ Automatic updates                                                                        |                                                                       |
+| ✅ Automatic sleep/wake                                                                     |                                                                       |
+
+### Earthly BYOC Satellites vs Earthly Self-Hosted Satellites
+
+| Similarities                                                     | Differences                                                                            |
+|------------------------------------------------------------------|----------------------------------------------------------------------------------------|
+| ✅ The satellite lives in your infrastructure, next to your tools | ❌ You cannot manually provision satellites                                             |
+| ✅ You pay for the usage the satellite incurs                     | ❌ You must allow Earthly access to at least some portion of an AWS account you control |
+|                                                                  | ❌ You pay the cloud provider, not Earthly, for the compute usage                       |
+|                                                                  | ❌ You are responsible for the Satellites monitoring and reliability                    |
```

Comment:
Reading through these tables again, I'm realizing that it's pretty hard to follow along. e.g. the difference "you are responsible for monitoring" isn't clear whether that refers to BYOC or self-hosted.

I think that these tables could be better represented like this:

### BYOC vs Cloud

| | Earthly Cloud | Earthly BYOC |
|-------|-------|-------|
| Who is responsible for monitoring and reliability of the Satellite | ✅ Earthly | ✅ Earthly |
| Satellites are deployed within your internal network | ❌ No | ✅ Yes |
| Earthly Cloud and Earthly staff are prevented from accessing your internal network | ✅ N/A | ✅ Yes |
| How is compute billed | ✅ Zero-margin compute via Earthly | ✅ Supported by you via your cloud provider |
| Automatic updates | ✅ Yes | ✅ Yes |
| Auto-sleep to drastically reduce compute cost | ✅ Yes | ✅ Yes |
| Automatic management and GCing of cache volumes | ✅ Yes | ✅ Yes |
| Users can launch and remove satellites via the `earthly sat` CLI | ✅ Yes | ✅ Yes |
| Requires access to a set of limited AWS capabilities | ✅ No | ❌ Yes |


### BYOC vs Self-Hosted

| | Earthly Self-Hosted | Earthly BYOC |
|-------|-------|-------|
| Who is responsible for monitoring and reliability of the Satellite | ❌ You | ✅ Earthly |
| Satellites are deployed within your internal network | ✅ Yes | ✅ Yes |
| Earthly Cloud and Earthly staff are prevented from accessing your internal network | ✅ Yes | ✅ Yes |
| Compute is billed directly to you from your cloud provider | ✅ Yes | ✅ Yes |
| Automatic updates | ❌ No | ✅ Yes |
| Auto-sleep to drastically reduce compute cost | ❌ No | ✅ Yes |
| Automatic management and GCing of cache volumes | ❌ No | ✅ Yes |
| Users can launch and remove satellites via the `earthly sat` CLI | ❌ No | ✅ Yes |
| Requires access to a set of limited AWS capabilities | ✅ No | ❌ Yes |


Markdown for this:

```
### BYOC vs Cloud

| | Earthly Cloud | Earthly BYOC |
|-------|-------|-------|
| Who is responsible for monitoring and reliability of the Satellite | ✅ Earthly | ✅ Earthly |
| Satellites are deployed within your internal network | ❌ No | ✅ Yes |
| Earthly Cloud and Earthly staff are prevented from accessing your internal network | ✅ N/A | ✅ Yes |
| How is compute billed | ✅ Zero-margin compute via Earthly | ✅ Supported by you via your cloud provider |
| Automatic updates | ✅ Yes | ✅ Yes |
| Auto-sleep to drastically reduce compute cost | ✅ Yes | ✅ Yes |
| Automatic management and GCing of cache volumes | ✅ Yes | ✅ Yes |
| Users can launch and remove satellites via the `earthly sat` CLI | ✅ Yes | ✅ Yes |
| Requires access to a set of limited AWS capabilities | ✅ No | ❌ Yes |


### BYOC vs Self-Hosted

| | Earthly Self-Hosted | Earthly BYOC |
|-------|-------|-------|
| Who is responsible for monitoring and reliability of the Satellite | ❌ You | ✅ Earthly |
| Satellites are deployed within your internal network | ✅ Yes | ✅ Yes |
| Earthly Cloud and Earthly staff are prevented from accessing your internal network | ✅ Yes | ✅ Yes |
| Compute is billed directly to you from your cloud provider | ✅ Yes | ✅ Yes |
| Automatic updates | ❌ No | ✅ Yes |
| Auto-sleep to drastically reduce compute cost | ❌ No | ✅ Yes |
| Automatic management and GCing of cache volumes | ❌ No | ✅ Yes |
| Users can launch and remove satellites via the `earthly sat` CLI | ❌ No | ✅ Yes |
| Requires access to a set of limited AWS capabilities | ✅ No | ❌ Yes |
```